mindsdb/integrations/handlers/pypi_handler/pypi_tables.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+import pandas as pd
+import pypistats
+from mindsdb_sql.parser import ast
+
+from mindsdb.integrations.libs.api_handler import APITable
+from mindsdb.utilities.log import get_log
+
+logger = get_log("integrations.pypi_handler")
+
+
+class PyPIRecentTable(APITable):
+    def select(self, query: ast.Select) -> pd.DataFrame:
+        df = pypistats.recent(self.handler.package, total=True, format="pandas")
```

Comment:
Maybe you were planning on doing this, but maybe the package to be viewed can be provided as part of the SQL query, i.e. the WHERE statement. For example:
```
SELECT *
FROM pypi_datasource.recent
WHERE package = 'pillow'
```

Reply:
@MinuraPunchihewa Then how's the implementation would be when there is no tables and `Handler.__init__()`?!

Reply:
Hey @lnxpy,
Apologies, I missed this. 

You will still need the handler and the tables.

The difference here would be that your `CREATE DATABASE` statements would be executed without passing any parameters. It will look something like this,
```
CREATE DATABASE pypi_datasource
WITH ENGINE = 'pypi'
```
Within the handler code, you will still have the `__init__()` method, but the difference is that we won't need to interact with the `connection_data` parameter in anyway.

I think this will be a good reference if you want to implement it this way,
https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/mediawiki_handler/mediawiki_handler.py